posts/2024-05-02-comparison-of-performance.ipynb

```diff
@@ -0,0 +1,521 @@
+{
```

Comment:
<p>Should there be a 'mojo' entry here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/coecms/coecms.github.io/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
I didn't get mojo to run. I've shortened the section even more, basically saying: Hey, there's that thing that might be useful in a few years.